app/grants/models.py

```diff
@@ -406,13 +424,13 @@ def get_debug_info(self):
                 error_reason = "insufficient_allowance"
 
             debug_info = f"""
-error_reason: {error_reason}
-==============================
-decimals: {decimals}
-balance: {balance}
-allowance: {allowance}
-amount_per_period: {self.amount_per_period}
-"""
+                error_reason: {error_reason}
```

Comment:
this will incorrectly create a bunch of tabs in the comments in the subscription object.